CONTRIBUTING.md

```diff
@@ -193,28 +235,98 @@ Designation basic format as an example:
+| green (logo)	| ![#9ac155](https://placehold.co/15x15/9ac155/9ac155.png) `#9ac155`	| `rgb(154, 193, 85)`	|							|
+| dark blue		| ![#95a4b3](https://placehold.co/15x15/95a4b3/95a4b3.png) `#95a4b3`	| `rgb(149, 164, 179)`	| Default color for icons	|
+| red			| ![#d11d1d](https://placehold.co/15x15/d11d1d/d11d1d.png) `#d11d1d`	| `rgb(209, 29, 29)`	| Error, Fault				|
+| yellow		| ![#ffd817](https://placehold.co/15x15/ffd817/ffd817.png) `#ffd817`	| `rgb(255, 216, 23)`	| Warning					|
+
+
+## Interactive-Flex-Query Importer
+
+An XML-compliant document is a file that is structured and formatted according to the rules of the eXtensible Markup Language (XML). It consists of a set of tags and attributes that contain information about the structure and content of the document.
+
+A valid XML document must meet the following conditions:
+
+* It must have a root element that contains all other elements.
+* Each element must be properly closed.
+* Attributes must be defined within a start tag and their values must be specified within quotes.
+* All elements must be properly nested, i.e. they must contain each other and not overlap.
+* There must be no comments or processing instructions outside the root element.
 
-## PDF Importer
+### Source location
+
+Interactive-Flex-Query importer: `name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/ibflex/`
+
+Test cases: `name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/ibflex/`
+
+
+### Imported transactions
+
+Imported transactions are processed in the same way as the [PDF importer](#imported-transactions-1).
+
+
+### Naming conventions for detected elements
+
+The importer extracts the following elements:
+
+* `type` --> The transaction type e.g. buy, sell, deposit, removal, dividends, taxes, fees
+* `buySell` --> The portfolio transaction type e.g. buy, sell.
+* `symbol` --> The symbol of the instrument traded.
+* `isin` --> International Securities Identification Number
+* `conid` --> The conid of the instrument traded. Same as the security code number (wkn).
+* `cusip` --> The CUSIP of the instrument traded. Same as the security code number (wkn).
+* `quantity` --> The number of units for the transaction.
+* `multiplier` --> The multiplier of the contract traded.
+* `proceeds` --> Calculated by multiplying the quantity and the transaction price. The proceeds figure will be negative for buys and positive for sales.
+* `reportDate` --> The date of the statement.
+* `tradeDate` --> The date of the execution.
+* `tradeTime` --> The time of the execution.
+* `dateTime` --> The date and time of the statement.
+* `netCash` --> Net cash is calculated by adding the proceeds plus tax plus commissions. (buy and sell)
+* `amount` --> Amount for account transactions
+* `currency` --> Currency of the total amount.
+* `fxRateToBase` --> The conversion rate from asset currency to base currency.
```

Comment:
Again, I do not think the belongs into a contribution guide. If at all, it belongs to the IBFlex class. But even then, let's be careful to copy documentation which will soon become outdated. The link to the IBK guides sounds good enough to me

Reply:
BTW, if you feel that is valuable, then we can also add a README.md to the "pdf" or "ibflex" folders